client/dashboard/app/style.scss

```diff
@@ -36,7 +36,7 @@ a {
 	font-size: $font-size-medium;
 }
 
-h1 {
+h1:not( .components-modal__header-heading ) {
```

Comment:
Not really happy about this change, any better ideas?

Reply:
Hmm... maybe move these existing CSS to https://github.com/Automattic/wp-calypso/blob/0a063059234e6cccc1f61843ecbe34e1b8cfb97f/client/dashboard/components/page-header/style.scss instead? 🥲

Reply:
I think that's a fair change. Updated in c404fa475b71b5750ff34b40d7a1fcdb7c787c35.

Reply:
we're using an h1 in a modal, is that really what we want? Aren't we supposed to have only one h1 in a page?

Reply:
Ok, I see the message about modal forcing h1. That's probably fair.

The downside I see with the change to move to page header, is that the page-header is probably a reusable component that can be extracted to the components package, and defining the h1 styles seems like an "application layer" thing that the reusable component should be agnostic about.

That said, it's not a big deal at the moment, there are multiple solutions here including a "theme" package for DS. The change is fine for now.

Reply:
Unsure of the semantics, but yes I could personally see all headings being subject to branding on the font level. I think there's a technical/component reusability question to answer as to how to best do this.